radio/src/gui/212x64/radio_hardware.cpp

```diff
@@ -115,14 +113,16 @@ void menuRadioHardware(event_t event)
         break;
       }
 #if defined(PCBX9E)
+      case ITEM_RADIO_HARDWARE_LS:
```

Comment:
But then the name of LS and RS is not anymore configurable on X9D / X9D+
I suggest only the 2 case lines for LS2 and RS2 are surrounded by the ```#if defined```